distribution/pull_v2.go

```diff
@@ -573,13 +590,15 @@ func (p *v2Puller) pullSchema2(ctx context.Context, ref reference.Named, mfst *s
 	}
 	defer release()
 
-	configJSON, unmarshalledConfig, err = receiveConfig(configChan, errChan)
-	if err != nil {
-		return "", "", err
-	}
+	if runtime.GOOS != "windows" {
```

Comment:
This could be `if configJSON == nil` so we don't have to worry about updating the conditional in two places.